content/docs/pulumi-cloud/admin/self-hosted/airgapped.md

```diff
@@ -0,0 +1,131 @@
+* **Industrial and Manufacturing**: Protecting critical infrastructure from cyber threats.
+
+Pulumi Self-Hosted offers a robust solution for these environments, enabling infrastructure automation without requiring external network dependencies.
+
+## Preparing for an Air-Gapped Deployment
+
+Here are the general areas to consider when preparing to use Pulumi in an air-gapped environment:
+
+* Obtain Pulumi Self-Hosted
+* Provision Internal Infrastructure and Install Pulumi Cloud
+* Create and Configure Artifact Mirrors
+
+### Obtain Pulumi Self-Hosted
+
+Ensure you have access to Pulumi Self-Hosted, which includes the Pulumi API and backend services. If you don't have
+access, you can either [contact us](/contact) or [request access here](/product/self-hosted/#self-hosted-trial).
+
+### Provision Internal Infrastructure and Install Pulumi Cloud
+
+You must set up internal servers for running Pulumi Self-Hosted, such as Kubernetes or virtual machines, as well as the necessary data stores.
+
+The complete infrastructure required typically includes:
+
+* *An Isolated Compute Environment*: A virtual machine (VM) or Kubernetes cluster within the air-gapped network.
+* *Pulumi Self-Hosted Installation Artifacts*: These can be retrieved from a network-accessible environment and transferred to the air-gapped system.
+* *A Private Container Registry*: Required to store Pulumi service images for deployment.
+* *Database and Storage Backend*: PostgreSQL and object storage (such as MinIO or an on-premises S3-compatible storage system) for state management.
+* *Internal Package Management*: To host Pulumi SDKs and required language runtimes, as external package managers (npm, PyPI, etc.) won't be accessible.
+* *Automation and CI/CD Setup*: Configured to run within the air-gapped network for secure infrastructure deployments.
+
+Pulumi supports many [deployment options](/docs/pulumi-cloud/admin/self-hosted/deployment-options) so it is a good idea to consult with your Pulumi representative on what approach will best meet your needs.
+
+### Create and Configure Artifact Mirrors
+
+Since the environment lacks internet access, you must provide local mirrors for certain artifacts typically downloaded over the internet:
+
+* Pulumi CLI and SDK packages (Node.js, Python, Go, .NET, etc.).
+* Pulumi provider binaries.
+* Container registries for Pulumi Cloud Docker images.
+
+You must configure all tools, scripts, and automation to use these mirrors instead of the default public internet URLs.
+
+## Step-by-Step Deployment Instructions
+
+Here are more detailed instructions on how to install Pulumi Self-Hosted to your air-gapped environment once it has been prepared:
+
+### Step 1: Deploy the Pulumi Self-Hosted Backend
+
+Pulumi Self-Hosted can be installed using Kubernetes, Docker, or virtual machines. In an air-gapped setup, follow these steps:
+
+1. Download the Pulumi Self-Hosted release
```

Comment:
Add a reference to where to get the images:

`[self-hosted images](/docs/pulumi-cloud/admin/self-hosted/components)`